testSrc/com/intellij/plugins/haxe/ide/HaxeCompilerErrorParsingTest.java

```diff
@@ -73,7 +74,9 @@ public void testNMEErrorAbsoluteUnix() {
 
     assertNotNull(compilerError);
     assertEquals(CompilerMessageCategory.ERROR, compilerError.getCategory());
-    assertEquals("/an/absolute/path/HelloWorld.hx", compilerError.getPath());
+    if (Platform.current() == Platform.UNIX) {
+      assertEquals("/an/absolute/path/HelloWorld.hx", compilerError.getPath());
```

Comment:
Is there a Windows equivalent?  Such as:
  assertEquals("C:\an\absolute\path\HelloWorld.hx", compilerError.getPath());

(Sorry for the duplicate comment, I need to stop commenting on the individual commits.)